cypress/support/fragments/data_import/job_profiles/jobProfileEdit.js

```diff
@@ -17,4 +17,15 @@ export default {
       .click();
     cy.do(Modal({ id: 'unlink-job-profile-modal' }).find(Button('Unlink')).click());
   },
+  unlinkMatchProfile: (number) => {
+    cy.get('[id*="branch-ROOT-MATCH-editable"]').eq(number).find('button[icon="unlink"]').click();
+    cy.do(Modal({ id: 'unlink-job-profile-modal' }).find(Button('Unlink')).click());
+  },
+  unlinkNonMatchProfile: (number) => {
+    cy.get('[id*="branch-ROOT-NON_MATCH-editable"]')
```

Comment:
Please use Interactors

Reply:
what interactor i can use for this element [id*="branch-ROOT-MATCH-editable"]?